scripts/package.ps1

```diff
@@ -29,9 +29,17 @@ foreach ($file in [System.IO.Directory]::EnumerateFiles("$repoRoot\src", "System
     }
 }
 
+Ensure-Nuget-Exists
+Write-Host "** Creating BrotliNative NuGet packages. **"
+$brotliExternalFile = "$repoRoot\external\BrotliNative.nuspec"
+Invoke-Expression "$nugetPath pack $brotliExternalFile -Version 0.0.1 -o $packagesPath"
```

Comment:
or even just a general-purpose external/*.nuspec for other packages like this in the future

https://github.com/dotnet/corefxlab/blob/master/scripts/package.ps1#L32-L35